addons/luamodel/luamodel.h

```diff
@@ -261,14 +262,69 @@ namespace Addons {
 RBDL_DLLAPI
 bool LuaModelReadFromFile (
   const char* filename,
-  Model* model,
+  Model* updModel,
   bool verbose = false);
 
 /** \brief Reads a model file and returns the names of all constraint sets.
  */
 RBDL_DLLAPI
 std::vector<std::string> LuaModelGetConstraintSetNames(const char* filename);
 
+/**
+  This function will load named body-fixed motion capture markers that are
+  attached to each body
+
+  @param filename: name of the lua file
+  @param model: reference to the (loaded) multibody model.
+  @param pointSet: an empty std::vector of Point structure
+  @param verbose: information will be printed to the command window if this
+                  is set to true.
+
+*/
+RBDL_DLLAPI
+bool LuaModelReadMotionCaptureMarkers (
+      const char* filename,
+      const Model* model,
+      std::vector<Point> &markerSet,
```

Comment:
naming convention: use `marker_set` instead of `markerSet`

Reply:
All of the camel-case names have been updated to snake-case (including the doxygen).